occurrences/tests/test_api.py

```diff
@@ -38,6 +44,38 @@ def autouse_db(db):
     pass
 
 
+STUDY_LEVELS_QUERY = """
+    query StudyLevels{
+        studyLevels {
+            edges {
+                node {
+                    id
+                    label
+                    level
+                    translations {
+                        languageCode
+                        label
+                    }
+                }
+            }
+        }
+    }
+"""
+
+STUDY_LEVEL_QUERY = """
+    query StudyLevel($id: ID!){
+        studyLevel(id: $id){
+            id
+            label
+            level
+            translations {
+                languageCode
+                label
+            }
+        }
+    }
+"""
+
```

Comment:
These needs to be updated in this frontend pull request https://github.com/City-of-Helsinki/palvelutarjotin-ui/pull/66